test/easyconfigs/easyconfigs.py

```diff
@@ -681,6 +681,10 @@ def template_easyconfig_test(self, spec):
                         # it should *not* be the same as the parent toolchain
                         self.assertNotEqual(dumped_dep[3], (orig_toolchain['name'], orig_toolchain['version']))
 
+        # take into account that for some string-valued easyconfig parameters (configopts & co),
+        # the easyblock may have injected additional values, which affects the dumped easyconfig file
+        elif isinstance(orig_val, basestring):
+            self.assertTrue(dumped_val.startswith(orig_val))
```

Comment:
this is required to avoid that the tests trip over the fact that `PythonPackage` adds additional options into `installopts` when `pip` is used, which affects the `installopts` value in the dump easyconfig file compared to the original `installopts` value specified in the `CVXOPT` easyconfig...